third_party/sycl/crosstool/computecpp.tpl

```diff
@@ -78,111 +66,19 @@ def get_device_compiler_flags(compiler_flags):
   ]
   return compiler_flags + computecpp_flags
 
-def remove_unknown_computecpp_flags(flags):
```

Comment:
This function wasn't use with the driver so I removed it. Are we sure we don't need it anymore? It compiles fine on my machine but not too sure for others.